pandas-stubs/core/series.pyi

```diff
@@ -1950,67 +1958,67 @@ class Series(IndexOpsMixin[S1], NDFrame):
         self,
         other: num | _ListLike | Series[S1],
         level: Level | None = ...,
-        fill_value: float | None = ...,
+        fill_value: float | None = None,
         axis: AxisIndex | None = ...,
     ) -> Series[S1]: ...
     def ne(
         self,
         other: Scalar | Series[S1],
         level: Level | None = ...,
-        fill_value: float | None = ...,
+        fill_value: float | None = None,
         axis: AxisIndex = ...,
     ) -> Series[_bool]: ...
     @final
-    def nunique(self, dropna: _bool = ...) -> int: ...
+    def nunique(self, dropna: _bool = True) -> int: ...
     def pow(
         self,
         other: num | _ListLike | Series[S1],
         level: Level | None = ...,
-        fill_value: float | None = ...,
+        fill_value: float | None = None,
         axis: AxisIndex | None = ...,
     ) -> Series[S1]: ...
     def prod(
         self,
         axis: AxisIndex | None = ...,
```

Comment:
For all of these operations, the `axis` value defaults to 0.  There are lots of them in here - I'm commenting on this one, but you should fix the others.  I think it is the case that if `axis` is an argument to a `Series` method, the default value is 0, because that's the only value that makes sense.